src/language-server.ts

```diff
@@ -37,37 +54,65 @@ export class LanguageServerAPI {
                 );
             });
 
+        this.client.onDidChangeState((e: StateChangeEvent) => {
+            switch(e.newState){
+                case 2:
+                    this.running = true
+                    break
+                default:
+                    this.running = false
+                    break
+            }
+        })
+
         const clientDisposable = this.client.start();
         ctx.subscriptions.push(clientDisposable);
     }
 
+    async initAccountManager() {
+        return this.client.sendRequest("workspace/executeCommand", {
+            command: INIT_ACCOUNT_MANAGER,
+            arguments: []
+        })
+    }
+
+    async changeEmulatorState(emulatorState: EmulatorState) {
+        return this.client.sendRequest("workspace/executeCommand", {
+            command: CHANGE_EMULATOR_STATE,
+            arguments: [
+                emulatorState
+            ]
+        })
+    }
+
     // Sends a request to switch the currently active account.
-    async switchActiveAccount(accountAddr: string) {
+    async switchActiveAccount(account: {name: string, address: string}) {
```

Comment:
```suggestion
    async switchActiveAccount(account: Account) {
```

I would use the `Account` type here

Reply:
Just re-commenting here in case you missed it